priv/static/index.html

```diff
@@ -44,7 +44,14 @@
           <div class="col-md-2">
           </div>
           <div class="col-md-8">
-            <input type="text" id="siteSearch" class="input-lg" placeholder="Enter a website you love..." autofocus>
+            <input type="url" id="siteSearch" class="input-lg" placeholder="Enter a website you love..." list="defaultURLs" autofocus>
```

Comment:
`type="url"` makes it so that my mobile browser will autocorrect a bit smarter (and I can hold down `.` key to autocomplete `.com | .net | .gov | etc...`